build-system/tasks/check-links.js

```diff
@@ -72,9 +73,10 @@ function checkLinks() {
     });
     if (deadLinksFound) {
         util.log(
-            util.colors.red('ERROR'), 'Dead links found. Please update',
-            util.colors.magenta(filesWithDeadLinks.join(',')));
-       process.exit(1);
+            util.colors.yellow('WARNING'),
+            'Dead links potentially found. Please update',
+            util.colors.magenta(filesWithDeadLinks.join(',')),
+            'if necessary.');
```

Comment:
Why have this warning message display twice? Looks like once for each file and once overall?